tests/src/Common/hostpolicymock/HostpolicyMock.cpp

```diff
@@ -14,7 +14,17 @@
 typedef wchar_t char_t;
 typedef std::wstring string_t;
 
-#else //!_Win32
+// Only create undecorated exports on Windows x86
+#if defined _X86_
+
+// Define undecorated exports to ease test set up from native code
```

Comment:
Where it the native code that this talks about?

I can see these used only by DllImport from C# that should not need this.

Reply:
In `CoreShim.cpp` - https://github.com/dotnet/coreclr/pull/22317/files#diff-9e0a551287730dea4e71a6728b0fded1R190

Reply:
ok